From 53234f03909f9367571a2c42e9538e1499aefc8a Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Wed, 6 Feb 2019 09:34:31 +0100
Subject: [PATCH] Add identification led

imx8m-var-dart-common.dtsi: Add LED2 as identification led

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@resin.io>

---
 arch/arm64/boot/dts/variscite/imx8m-var-dart-common.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/variscite/imx8m-var-dart-common.dtsi b/arch/arm64/boot/dts/variscite/imx8m-var-dart-common.dtsi
index a52401c..4d1a099d 100644
--- a/arch/arm64/boot/dts/variscite/imx8m-var-dart-common.dtsi
+++ b/arch/arm64/boot/dts/variscite/imx8m-var-dart-common.dtsi
@@ -145,6 +145,11 @@
 			gpios = <&gpio4 17 GPIO_ACTIVE_HIGH>;
 			linux,default-trigger = "mmc0";
 		};
+
+		led2 {
+			label = "led2";
+			gpios = <&gpio4 14 GPIO_ACTIVE_HIGH>;
+		};
 	};
 };
 
-- 
2.7.4

